content/en/docs/install-operate/download.md

```diff
@@ -5,21 +5,30 @@ aliases: [/docs/getting-started/download]
 weight: 20
 ---
 
-The Falco Project community only supports two ways for downloading and running Falco:
+The Falco Project supports two ways for downloading and running Falco:
 
- - Running Falco directly on a Linux host
- - Running the Falco userspace program in a container, with a driver installed on the underlying host.
+ - Running Falco directly on a Linux host.
+ - Running Falco in a container.
```

Comment:
[nit] can be added in later somehow: I often get asked if you have to run Falco in every container etc. Perhaps something in line of: 

No matter what deployment strategy is chosen for Falco, it only needs to run once per Linux operating system, whether it's on a bare-metal server or a virtual machine (VM). Falco takes advantage of the fact that the kernel is shared for containerized infrastructures like Kubernetes, allowing it to monitor the behavior of all types of workloads, both host, and container workloads. Deploying Falco within a container with the appropriate host mounts is, therefore, equivalent to deploying Falco directly on the Linux host.

Reply:
This is great feedback. Anyway, I have no idea where to add it :thinking: (for sure, not in the download page) @LucaGuerra ?

Reply:
Maybe before Kubernetes 

https://falco.org/docs/install-operate/deployment/

Also seems like a page called "deployment" and only talking about Kubernetes is missing a point anyways that you can run it on any Linux, bare-metal or VM.

We could just add that before the "Kubernetes" header for now and check back later?

Reply:
@leogr I have added this in the suggestions in a new PR https://github.com/falcosecurity/falco-website/pull/1113